config/core/test-configs.yaml

```diff
@@ -2327,6 +2327,21 @@ test_configs:
   - device_type: r8a774a1-hihope-rzg2m-ex
     test_plans:
       - baseline
```

Comment:
For some reason I can't see `baseline` results for this platform on staging.  It's worth checking if the tests are actually submitted, maybe it's a problem with the buildroot image.

Reply:
is this machine ? https://lava.ciplatform.org/scheduler/device_type/r8a774a1-hihope-rzg2m-ex ?

Reply:
i see baseline results
like this: https://lava.ciplatform.org/scheduler/job/525406

Reply:
Yes that's already in production.  This PR adds baseline-nfs.